xml/art_sle_ha_pmremote.xml

```diff
@@ -53,16 +53,28 @@
   </para>
   <itemizedlist>
    <listitem>
-    <para>A cluster node running &productname; 12 GA or higher. In this guide,
-     its host name is <systemitem class="domainname">&node1;</systemitem>.</para>
+    <para>
+     Two cluster nodes running &productname; 12 GA or higher. In this guide,
+     their host names are <systemitem class="domainname">&node1;</systemitem>
+     and <systemitem class="domainname">&node2;</systemitem>.
+    </para>
    </listitem>
    <listitem>
-    <para>Failover of resources from one node to the other if the active host
-     breaks down (active/passive setup).</para>
+    <para>
+     Failover of resources from one node to the other if the active host
+     breaks down (active/passive setup).
+    </para>
    </listitem>
    <listitem>
-    <para>Virtual guest nodes running &pmremote;. In this guide, one guest
-     node is named <systemitem class="domainname">&wsII;</systemitem>.
+    <para>
+     Virtual guest nodes running &pmremote;. In this guide, one guest
+     node is named <systemitem class="domainname">&node4;</systemitem>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     One physical remote node running &pmremote;. In this guide, one guest
+     node is named <systemitem class="domainname">&node3;</systemitem>.
```

Comment:
Should we probably put this before the previous point "A virtual guest node..." to have the consistent ordering?
And how about somehow put an "or" between these two points?